src/client/scenes/universescene.cpp

```diff
@@ -128,6 +128,7 @@ void conquerspace::scene::UniverseScene::Update(float deltaTime) {
     if (GetApp().GetUniverse().ToTick() && !game_halted) {
         // Game tick
         simulation->tick();
+        system_renderer->SeeEntity();
```

Comment:
Don't do this, this will reload the terrain every time. Make `SeeEntity()` private, and make a recenter function.